### PR TITLE
Security feature email alert

### LINKED ISSUE #128: Security feature email alert
A email alert to me if someone else is trying to login to my account

### PR DESCRIPTION
**Pull request by arpit456jain**
# Description

Security feature email alert

Fixes #128 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: bookstore/client/views.py [hunk 1]
```diff
@@ -46,6 +46,17 @@ def check(email):
...
 mail = Mail(app)
 
+def send_warning_email(email):
+    import smtplib
+    con = smtplib.SMTP("smtp.gmail.com",587)
+    con.ehlo()
+    con.starttls()
+    admin_email = "admin_email"
+    admin_password = "admin_password"
+    con.login(admin_email,admin_password)
```
Comment (arpit456jain) [inline bookstore/client/views.py:56]: @Akshay6890 don't forgot to insert a email from which email will send .

After that you can check this PR just try login in to your account with wrong password more than 5 times

**Comment by arpit456jain** [comment]
@Akshay6890 @vikasdo kindly review this PR

File: bookstore/client/views.py (hunk 1 above)
Comment (Akshay6890) [reply bookstore/client/views.py:56]: > @Akshay6890 don't forgot to insert a email from which email will send .
> 
> After that you can check this PR just try login in to your account with wrong password more than 5 times

that email variable will store the email right?

Comment (arpit456jain) [reply bookstore/client/views.py:56]: no

Comment (arpit456jain) [reply bookstore/client/views.py:56]: this variable is for the email from which the email has to be send @Akshay6890

Comment (Akshay6890) [reply bookstore/client/views.py:56]: > no

so we have to hard code the email?

Comment (arpit456jain) [reply bookstore/client/views.py:56]: in short .
 email and password of admin or owner of website @Akshay6890

Comment (Akshay6890) [reply bookstore/client/views.py:56]: > in short .
> email and password of admin or owner of website @Akshay6890

yea, I get it, my bad 😅

Comment (arpit456jain) [reply bookstore/client/views.py:56]: is there any variable for email and  password @Akshay6890

Comment (Akshay6890) [reply bookstore/client/views.py:56]: > is there any variable for email and password @Akshay6890

we used to create sessions right?

Comment (Akshay6890) [reply bookstore/client/views.py:56]: > is there any variable for email and password @Akshay6890

we assigned those values when user logs in

Comment (Akshay6890) [reply bookstore/client/views.py:56]: > is there any variable for email and password @Akshay6890

of course only email

**Pull request merged**